File: app/src/main - 副本/java/com/zmcursor/mydaliy/Model/Bean/TopStory.java
package com.zmcursor.mydaliy.Model.Bean;

/**
 * Created by ZMcursor on 2018/5/30 0030.
 */

public class TopStory extends BaseStory {

    public String image;

    @Override
    public String getImageUrl() {
        return image;
    }

    @Override
    public String toString() {
        return "TopStory{" +
                "type=" + type +
                ", id=" + id +
                ", ga_prefix=" + ga_prefix +
                ", multipic=" + multipic +
                ", title='" + title + '\'' +
                ", image='" + image + '\'' +
                '}';
    }

}

File: app/src/main/java/com/zmcursor/mydaliy/Model/Bean/TopStory.kt
package com.zmcursor.mydaliy.Model.Bean

/**
 * Created by ZMcursor on 2018/5/30 0030.
 */

class TopStory : BaseStory() {

    var image: String? = null

    override var imageUrl: String? = null
        get() = image

    override fun toString(): String {
        return "TopStory{" +
                "type=" + type +
                ", id=" + id +
                ", ga_prefix=" + ga_prefix +
                ", multipic=" + multipic +
                ", title='" + title + '\'' +
                ", image='" + image + '\'' +
                '}'
    }

}
